packages/patrol/lib/src/custom_finders/patrol_integration_tester.dart

```diff
@@ -1,17 +1,22 @@
 import 'package:patrol/src/native/native_automator.dart';
 import 'package:patrol/src/native/native_automator2.dart';
 import 'package:patrol_finders/patrol_finders.dart' as finders;
+import 'package:patrol_log/patrol_log.dart';
 
 /// PatrolIntegrationTester extends the capabilities of [finders.PatrolTester]
 /// with the ability to interact with native platform features via [native].
 class PatrolIntegrationTester extends finders.PatrolTester {
   /// Creates a new [PatrolIntegrationTester] which wraps [tester].
-  const PatrolIntegrationTester({
+  PatrolIntegrationTester({
     required super.tester,
     required super.config,
     required this.nativeAutomator,
     required this.nativeAutomator2,
-  });
+    required PatrolLogWriter patrolLog,
```

Comment:
This one will sort of break the API (look here to [semver check](https://github.com/leancodepl/patrol/actions/runs/11615023075/job/32344576266?pr=2387). I know most of the users are not using this class directly but some of the projects do. Can we somehow leave this `PatrolLogWriter` as an optional parameter? At least until we introduce patrol 4.0

Reply:
I've change it, so it won't be passed as argument, but initialized locally